code/datums/gamemode/dynamic/dynamic_rulesets_latejoin.dm

```diff
@@ -8,7 +8,8 @@
 /datum/dynamic_ruleset/latejoin/infiltrator
 	name = "Syndicate Infiltrator"
 	role_category = ROLE_TRAITOR
-	restricted_from_jobs = list("Cyborg","Mobile MMI","Security Officer", "Warden", "Detective", "Head of Security", "Captain")
+	protected_from_jobs = list("Security Officer", "Warden", "Head of Personnel", "Detective", "Head of Security", "Captain")
+	restricted_from_jobs = list("AI","Cyborg","Mobile MMI")
```

Comment:
:thinking:

Reply:
An AI can't latejoin

Reply:
Well, you can't really latejoin as those anyway

Reply:
But you can, as long an admin frees the slot.

Reply:
Last I heard, doing that was a hard "no." Has it changed with datums or something?
Also even if you could do it, it would be a bit of an exception more than a rule

Reply:
I'm pretty sure that, just like trader, you can't latejoin as silicons, no matter what.
Where would they spawn?

Reply:
It just creates an assistant with an "AI" job title, no PDA, and no access in the arrival shutle @Kammerjunk @Exxion

Reply:
You can latejoin as mommi/borg just fine. Not sure about AI since i never tried it

Reply:
Traders can latejoin now